packages/eas-cli/src/graphql/queries/BuildQuery.ts

```diff
@@ -0,0 +1,75 @@
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../client';
+import { Build } from '../types/Build';
+
+type Filters = Partial<Pick<Build, 'platform' | 'status'>> & {
+  order?: number;
+  limit?: number;
+};
+
+type ArtifactFragmentType = Pick<Build, 'artifacts'>;
+type PlatformAndArtifactFragmentType = Pick<Build, 'platform' | 'artifacts'>;
+
+export class BuildQuery {
+  static async forArtifactByIdAsync(buildId: string): Promise<PlatformAndArtifactFragmentType> {
+    const data = await withErrorHandlingAsync(
+      graphqlClient
+        .query<{ builds: { byId: PlatformAndArtifactFragmentType } }>(
+          gql`
+          {
+            builds {
+              byId(buildId: "${buildId}") {
+                platform,
+                artifacts {
+                  buildUrl
+                }
+              }
+            }
+          }`
```

Comment:
Rather than string interpolation, please use [variables](http://spec.graphql.org/draft/#sec-Language.Variables) to pass parameters like this:
```suggestion
          gql`
          query ($buildId: ID!) {
            builds {
              byId(buildId: $buildId) {
                platform
                artifacts {
                  buildUrl
                }
              }
            }
          }`, { buildId }
```
This feature is built in the GraphQL language to avoid piecing queries together from strings. It also ensures the queries are maximally reusable and is less error prone when it comes to things like correctly escaping the variables.